ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -137,12 +144,12 @@ private[apiserver] final class ApiUserManagementService(
               resourceVersionO = pResourceVersion,
               annotations = pAnnotations,
             ),
-            identityProviderId =
-              IdentityProviderId.Default, // TODO IDP: replace with the value coming from API
+            identityProviderId = identityProviderId,
           ),
           pFieldMask,
         )
       } { case (user, fieldMask) =>
+        // TODO IDP: Check if user belongs to the same `identityProviderId` or is ParticipantAdmin
```

Comment:
left-over todo?

Reply:
The big story of supporting IDPs is split into smaller tickets, otherwise it would be 4000+ lines change. The actual authorization rule change will come next which will use the IDP provided by the user.